Resources/config/doctrine/BaseGallery.orm.xml

```diff
@@ -3,7 +3,7 @@
     <mapped-superclass name="Sonata\MediaBundle\Entity\BaseGallery">
         <field name="name" type="string" column="name" length="255"/>
         <field name="context" type="string" column="context" length="64"/>
-        <field name="defaultFormat" type="string" column="default_format" length="255"/>
+        <field name="defaultFormat" type="string" column="default_format" nullable="true" length="255"/>
```

Comment:
We set it this field as nullable or we set `reference` as default value. In this second case we don't need the `if` in `getDefaultFormat` anymore.